src/Seed/Result/LineComplexity.cs

```diff
@@ -0,0 +1,12 @@
+using Newtonsoft.Json;
+
+namespace CodacyCSharp.Metrics.Seed.Result
+{
+    public sealed class LineComplexity
+    {
+		[JsonProperty (PropertyName = "line")]
+        public long Line { get; set; }
```

Comment:
Everything is aligned with tabs, except for this couple of getters and setters

Tabs vs spaces is a discussion on itself, but we could make it consistent :)

Reply:
I usually use a tool for checking `.editorconfig` file across the project, but it escaped me. It should be fine now.